lib/routes/hazyresearch/blog.ts

```diff
@@ -0,0 +1,101 @@
+import { Route } from '@/types';
```

Comment:
[Namespace should be](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) the same as the secondary domain name `stanford`.